src/components/reset-password-2fa-auth-app/tests/reset-password-2fa-auth-app-controller.test.ts

```diff
@@ -111,7 +111,7 @@ describe("reset password 2fa auth app controller", () => {
       req.session.user = {
         email: "joe.bloggs@test.com",
         reauthenticate: "1234",
-        wrongCodeEnteredLock: futureDate,
+        wrongCodeEnteredPasswordResetMfaLock: futureDate,
       };
 
       await resetPassword2FAAuthAppGet()(req as Request, res as Response);
```

Comment:
This test looks like it's missing an assertion (existing problem with the code but we should fix it)